proto/ibc/applications/transfer/v2/packet.proto

```diff
@@ -16,4 +16,6 @@ message FungibleTokenPacketData {
   string sender = 3;
   // the recipient address on the destination chain
   string receiver = 4;
+  // optional metadata
+  bytes metadata = 5;
```

Comment:
Is the `metadata` something we should also include in the events emitted [here](https://github.com/cosmos/ibc-go/blob/main/modules/apps/transfer/ibc_module.go#L191-L209), [here](https://github.com/cosmos/ibc-go/blob/main/modules/apps/transfer/ibc_module.go#L235-L262) and [here](https://github.com/cosmos/ibc-go/blob/main/modules/apps/transfer/ibc_module.go#L282-L290). If yes, then we should also update [the docs](https://github.com/cosmos/ibc-go/blob/main/docs/apps/transfer/events.md#events).